pkg/networks/usernet/recoincile.go

```diff
@@ -143,7 +143,7 @@ func Stop(ctx context.Context, name string) error {
 			return err
 		}
 
-		if err := osutil.SysKill(pid, osutil.SigKill); err != nil {
+		if err := osutil.SysKill(pid, osutil.SigInt); err != nil {
```

Comment:
Can we fallback to SIGKILL on an error?

Reply:
`go func(){ Sleep(3 * second); SysKill(pid, SigKill) }()` may suffice, unless the PID is reused for another process within the 3 seconds.

Reply:
Let me postpone this PR to v2.0.1 (or later)